gobblin-core/src/main/java/gobblin/writer/AvroHdfsDataWriter.java

```diff
@@ -11,8 +11,14 @@
 
 package gobblin.writer;
 
+import com.google.common.base.Preconditions;
```

Comment:
Yeah it wasn't clear to me, since there are multiple conventions. I've added the import order we use to https://github.com/linkedin/gobblin/wiki/CodingStyle